src/styles/functions.js

```diff
@@ -29,7 +29,7 @@ const growOverParentPadding = (widthPercent) => css`
  * @property {function=} extraLarge
  */
 /** @type {mediaQueries} */
-const media = Object.keys(mediaSizes).reduce((acc, label) => {
+const Media = Object.keys(mediaSizes).reduce((acc, label) => {
   if (label === 'extraLarge') {
     acc[label] = (...args) => css`
```

Comment:
this change in variable name breaks something in our base components container file.

Reply:
It's due to naming convention warnings. Do you prefer to return them to the previous namings, or I update the code according to the new ones?